kombu/transport/kafka.py

```diff
@@ -0,0 +1,336 @@
+"""
+kombu.transport.kafka
```

Comment:
Well, for the 1st time we have green tests! Now is time to refactor the thing if needed. Lets start with the futures `absolute_import` and `unicode_literals`?